packages/shared/web/src/dashboard-view/hooks/use-liquidity-list.tsx

```diff
@@ -178,16 +178,16 @@ export const useLiquidityList = (
   const listData = yieldData
     .map((y) => {
       const walletBalance = account
-        ? account.balances.find((t) => t.tokenId === y.underlying.id)
```

Comment:
It seems to be like there is something wrong with the types here. If `y` is undefined then the entire return struct is basically empty

Reply:
Will address in my next PR